infra/src/main/resources/db/migration/prod/V39__add_projektivelho_integration.sql

```diff
@@ -0,0 +1,35 @@
+create type integrations.projektivelho_file_status as enum ('NOT_IM', 'IMPORTED', 'REJECTED', 'ACCEPTED');
+create type integrations.projektivelho_search_status as enum ('WAITING', 'FETCHING', 'FINISHED', 'ERROR');
+
+create table integrations.projektivelho_search
+(
+  id          int primary key generated always as identity,
+  status      integrations.projektivelho_search_status not null,
+  token       varchar(50),
+  valid_until timestamp with time zone not null
+);
```

Comment:
Yhden Velho-haun tiedot. Mietin vähän että olisiko näitäkin pitänyt siivoilla pois välillä, mutta toisaalta tällä hetkellä tänne tauluun tulee yksi rivi tunnissa, joka ei ole ihan kauheasti.